Rsupport/setupR.sh

```diff
@@ -85,7 +85,7 @@ if [ "x$BUILD_R" != "x" ]; then
 fi
 
 curl -sSL https://github.com/Crick-CancerGenomics/ascat/releases/download/v${VERSION_ASCAT}/ASCAT_${VERSION_ASCAT}.tar.gz > ascat_R.tar.gz
-curl -sSL https://github.com/cancerit/battenberg/archive/$VERSION_BB.tar.gz > BB_R.tar.gz
+curl -ssL https://github.com/cancerit/battenberg/archive/$VERSION_BB.tar.gz > BB_R.tar.gz
```

Comment:
This is wrong, it should be `-sSL`

```
       -S, --show-error
              When used with -s, --silent, it makes curl show an error message if it fails.

       -s, --silent
              Silent or quiet mode. Don't show progress meter or error messages.  Makes Curl mute. It will still output the data you ask for, potentially even to the terminal/stdout unless you redirect it.
````

Reply:
Wow good catch!

Reply:
It only stood out as there was no other reason for the line to have changed